doc/reference/gerbil/runtime/MOP.md

```diff
@@ -571,55 +483,196 @@ Looks up the method with *id* in *obj*'s hierarchy.
 Looks up the method with *id* in *obj*'s hierarchy and returns
 a procedure which applies the method currying the object.
```

Comment:
What are the arguments to the method returned? Does it include obj, or only the following arguments?

Reply:
no, the object is curried -- only the arguments.